### ISSUE TITLE
OSX support

### ISSUE THREAD

**Issue by kalliste** [akofman/cordova-plugin-add-swift-support#19]
Right now the plugin supports iOS apps only, but Cordova and Swift are also available for OSX

**Comment by akofman** [comment]
Did you have a try with this plugin in an OSX context ? What issues do you encounter ?
I'm not really comfortable with the OSX version of Cordova,  I'll use Electron in that case.

**Comment by kalliste** [comment]
It wasn't immediately obvious how I would need to update the plugin code to get it going under OSX so I haven't tried it yet. I was looking at using it because I am wrapping a (proprietary) library in a cordova plugin and the library works on OSX so it would be convenient to test the plugin code this way.

**Comment by jab** [comment]
Interested in the same, and just found this issue. Did either of you ever get a chance to look at this since the most recent comment ~8 months ago? Happy to try to help too (though we're using the https://github.com/bitjson/cordova-plugin-swift-support fork only because [that's the one that [cordova-plugin-qrscanner depends on](https://github.com/bitpay/cordova-plugin-qrscanner/blob/0cb180ec9931dfe56348bdb3def0a9eebf4599ee/plugin.xml#L38), no idea why they're apparently preferring to leave their fork divergent rather than attempting to merge upstream? /cc @bitjson -- any interest in merging back here, or explaining in your README the purpose of continuing to maintain a divergent fork? Thanks:)

**Comment by bitjson** [comment]
Hey @jab – info in this thread: https://github.com/akofman/cordova-plugin-add-swift-support/issues/24

TL;DR:
probably use this one, not my fork
(but my fork will stay as-is for https://github.com/bitpay/cordova-plugin-qrscanner)

**Comment by jab** [comment]
@bitjson That link to #24 is really helpful, thanks! (Just sent you a [PR](https://github.com/bitjson/cordova-plugin-swift-support/pull/4) to add that info to your README so others can find it more easily in the future.)

We'll probably stick to using your fork, since we already depend on cordova-plugin-qrscanner, and it seems better to not depend on both swift-support plugins as long as we can avoid it.

While we're here, do you have any interest in adding OS X support to your fork? (I'd ask that by opening a new issue for it in your repo, but it looks like issues are disabled there. Any interest in going into https://github.com/bitjson/cordova-plugin-swift-support/settings and enabling issues?)

Thanks again!

**Comment by jab** [comment]
Just a wanted to try pinging you again @bitjson and @akofman - would love to know each of your interest in collaborating on adding OS X support (even if only to merge a patch), thanks!

**Comment by akofman** [comment]
Hey,

I close this issue because I think #37 responded to your needs.

**Comment by kalliste** [comment]
#37 did not fix it but I just submitted a pull request that does

**Comment by kalliste** [comment]
Pull request is #46

**Comment by kalliste** [comment]
The resulting project doesn't build :/ but this gets us closer
